driver/server/server.go

```diff
@@ -27,16 +32,18 @@ var (
 
 // Server is a grpc server for the communication with the driver.
 type Server struct {
-	*GRPCServer
+	grpc *grpc.Server
 	// Logger a logger to be used by the server.
 	Logger Logger
 
 	d driver.DriverModule
+
+	closers []io.Closer
```

Comment:
Please document this field (I realize it's unexported, but the semantics are slightly subtle).